test/specs/repo_client/interwiki-links.js

```diff
@@ -1,33 +1,38 @@
 'use strict';
 
-const fs = require( 'fs' );
+const fsPromises = require( 'fs/promises' );
 const assert = require( 'assert' );
+const readFileEncoding = require( '../../helpers/readFileEncoding' );
 
 describe( 'Interwiki links', function () {
-
-	it( 'Should be able to insert interwiki links', function () {
-		const repoLink = fs.readFileSync( __dirname + '/interwiki-link.sql', 'utf8' )
+	it( 'Should be able to insert interwiki links', async () => {
+		const repoLinkFile = await fsPromises.readFile(
+			__dirname + '/interwiki-link.sql',
+			readFileEncoding.utf8
+		);
+		const repoLink = repoLinkFile
 			.replace( /<WIKI_ID>/g, 'client_wiki' )
 			.replace( /<HOSTNAME>/g, process.env.MW_CLIENT_SERVER );
-		browser.dbQuery( repoLink );
-		assert(
-			browser.dbQuery( 'SELECT iw_url FROM interwiki WHERE iw_prefix = "client_wiki"' )
-				.indexOf( process.env.MW_CLIENT_SERVER ) > -1
+		await browser.dbQuery( repoLink );
+		const clientWikiQueryResults = await browser.dbQuery(
+			'SELECT iw_url FROM interwiki WHERE iw_prefix = "client_wiki"'
 		);
+		assert( clientWikiQueryResults.indexOf( process.env.MW_CLIENT_SERVER ) > -1 );
 
 		const config = {
 			user: process.env.DB_USER,
 			pass: process.env.DB_PASS,
 			database: 'client_wiki'
 		};
-		const clientLink = fs.readFileSync( __dirname + '/interwiki-link.sql', 'utf8' )
+		const clientLink = repoLinkFile
 			.replace( /<WIKI_ID>/g, 'my_wiki' )
 			.replace( /<HOSTNAME>/g, process.env.MW_SERVER );
 
```

Comment:
```suggestion
		const clientLink = repoLink;
```

Reply:
I believe we want these values properly replaced in the SQL code